docker/run.sh

```diff
@@ -52,30 +52,44 @@ function run_tests() {
     setup $TEST
     delta=$((`date +%s` - $now))
 
-    send_metrics_to_datadog "setup" $delta
+    send_timing_metric_to_datadog "setup" $delta
 
     now=`date +%s`
     su cchq -c "../run_tests $TEST $(printf " %q" "$@")"
     delta=$((`date +%s` - $now))
 
-    send_metrics_to_datadog "tests" $delta
+    send_timing_metric_to_datadog "tests" $delta
+    send_counter_metric_to_datadog
 }
 
-function send_metrics_to_datadog() {
+function send_timing_metric_to_datadog() {
+    send_metric_to_datadog "travis.timings.$1" $2 "gauge"
```

Comment:
nice you're right. you have to edit the metadata of the metric in the metric summary page: https://app.datadoghq.com/metric/summary. this could very useful